### PR TITLE
fixes CC-806

### PR DESCRIPTION
**Pull request by wwwtyro**

### CODE REVIEW

**Comment by wwwtyro** [comment]
test this

**Comment by wplu** [comment]
retest this please

File: commons/docker/api.go [hunk 1]
```diff
@@ -556,7 +556,11 @@ func (img *Image) Tag(tag string) (*Image, error) {
 	}{img.UUID, img.ID.String(), iid.BaseName(), iid.Registry(), iid.Tag}
 
 	glog.V(1).Infof("tagging image %s as: %s", args.repo, args.tag)
-	err = dc.TagImage(args.name, dockerclient.TagImageOptions{Repo: args.repo, Tag: args.tag})
+	opts := dockerclient.TagImageOptions{Repo: args.repo, Tag: args.tag}
+	if iid.Tag == "latest" {
```
Comment (wplu) [inline commons/docker/api.go]: iid.Tag should probably be args.tag, but that's ok

**Comment by wplu** [comment]
ship - waiting for test build to pass

**Pull request merged**